File: eva-core/eva-core-common/src/main/java/io/nerv/properties/DataPermission.java
package io.nerv.properties;

import lombok.Data;

import java.util.List;

/**
 * 数据权限配置类
 */
@Data
public class DataPermission {
    /** 是否启用 **/
    private boolean enable;

    /** 需要排除得表 **/
    private List<String> excludeTables;

    /** 需要排除得语句 **/
    private List<String> excludeStatements;
}

File: eva-core/src/main/kotlin/io/nerv/properties/DataPermission.kt
package io.nerv.properties

/**
 * 数据权限配置类
 */
class DataPermission {
    /** 是否启用  */
    var enable = false

    /** 需要排除得表  */
    var excludeTables: List<String>? = null

    /** 需要排除得语句  */
    var excludeStatements: List<String>? = null
}